src/main/java/net/sf/jabref/gui/search/SearchResultFrame.java

```diff
@@ -460,36 +538,40 @@ public Object getColumnValue(BibEntry entry, int column) {
                     } else {
                         return null;
                     }
-                case URL_COL:
-                    if (entry.hasField(FieldName.URL)) {
-                        urlLabel.setToolTipText(entry.getFieldOptional(FieldName.URL).get());
+                case URL_COL: {
+                    Optional<String> urlField = entry.getFieldOptional(FieldName.URL);
```

Comment:
They are there to define the scope for the `width` variable which are both in `DATABASE_COL` and `default`.